tests/cxx/template_member_functions.cc

```diff
@@ -23,9 +23,23 @@ struct Tpl {
 class IndirectClass;
 using NonProviding = Tpl<IndirectClass>;
 
+template <typename T>
+struct Outer {
+  T t;
+};
+
+template <typename T>
+void FnUsingNestedTpl() {
+  Outer<Tpl<T>> o;
+}
+
 void Fn() {
   // IWYU: IndirectClass is...*indirect.h
   NonProviding::StaticFn();
+
+  // IndirectClass is not fully used here. For this test case, it is important
+  // that StaticFn is used in the translation unit to be instantiated.
```

Comment:
Can you just add a little clause to also describe _why_ `StaticFn` is important for this (seemingly-unrelated) construct?

Reply:
I'm looking for a `, so that ...` or `, because ...` -- hopefully there's a short rationale hiding.